cpp/src/parquet/encoding.cc

```diff
@@ -2173,7 +2201,8 @@ class DeltaBitPackDecoder : public DecoderImpl, virtual public TypedDecoder<DTyp
 
  private:
   void InitBlock() {
-    int32_t block_size;
+    // The number of values per block.
+    uint32_t block_size;
```

Comment:
I changed the types according to spec https://github.com/apache/parquet-format/blob/master/Encodings.md#delta-encoding-delta_binary_packed--5